assets/src/components/detoursTable.tsx

```diff
@@ -9,6 +9,25 @@ interface DetoursTableProps {
   data: SimpleDetour[]
 }
 
+export enum DetourStatus {
+  Draft = "draft",
+  Active = "active",
+  Closed = "closed"
+}
+
+export const timestampNameForStatus = (status: DetourStatus) => {
```

Comment:
I'd also like to do this
```suggestion
export const timestampLabelFromStatus = (status: DetourStatus) => {
```